crates/holochain_trace/src/lib.rs

```diff
@@ -157,12 +157,16 @@ impl FromStr for Output {
 /// Run logging in a unit test.
 ///
 /// RUST_LOG must be set or this is a no-op.
-pub fn test_run() -> Result<(), errors::TracingError> {
+pub fn test_run() {
```

Comment:
Here's the change to be reviewing, everything else is just a consequence of changing the signature for this function.